src/modules/Invoice/html_client/mod_invoice_index.html.twig

```diff
@@ -5,101 +5,109 @@
-                            <td>{{ invoice.total | money(invoice.currency) }}</td>
-                            <td><a href="{{ 'invoice'|link }}/{{ invoice.hash }}" class="btn btn-primary btn-small">{{ 'View'|trans }}</a></td>
-                        </tr>
-                        {% endfor %}
-                        {% else %}
-                        <tr>
-                            <td colspan="7">{{ 'The list is empty'|trans }}</td>
-                        </tr>
-                        {% endif %}
-                        </tbody>
-                    </table>
-
+                <div class="card-body">
+                    <nav>
+                        <div class="nav nav-tabs" id="invoice-tab" role="tablist">
+                            <button class="nav-link active" id="unpaid-tab" data-bs-toggle="tab"
+                                    data-bs-target="#nav-unpaid" type="button" role="tab" aria-controls="nav-unpaid"
+                                    aria-selected="true">{{ 'Unpaid'|trans }}{% if unpaid_invoices.total %}&nbsp;<span
+                                    class="badge bg-danger">{{ unpaid_invoices.total }}</span>{% endif %}</button>
+                            <button class="nav-link" id="paid-tab" data-bs-toggle="tab" data-bs-target="#nav-paid"
+                                    type="button" role="tab" aria-controls="nav-paid"
+                                    aria-selected="false">{{ 'Paid'|trans }}&nbsp;<span
+                                    class="badge bg-secondary">{{ paid_invoices.total }}</span></button>
+                        </div>
+                    </nav>
+                    <div class="tab-content" id="nav-tabContent">
+                        <div class="tab-pane fade show active table-responsive" id="nav-unpaid" role="tabpanel"
+                             aria-labelledby="nav-home-tab" tabindex="0">
+                            <table class="table table-bordered table-hover">
+                                <thead>
+                                <tr>
+                                    <th>{{ 'Title'|trans }}</th>
+                                    <th>{{ 'Invoice Date'|trans }}</th>
+                                    <th>{{ 'Due Date'|trans }}</th>
+                                    <th>{{ 'Total'|trans }}</th>
+                                    <th>{{ 'Actions'|trans }}</th>
+                                </tr>
+                                </thead>
+                                <tbody>
+                                {% if unpaid_invoices.total > 0 %}
+                                    {% for i, invoice in unpaid_invoices.list %}
+                                        <tr>
+                                            <td>{{ "Invoice #%05s"|format(invoice.id) }}</td>
+                                            <td>{{ invoice.created_at|format_date }}</td>
+                                            <td>{{ invoice.due_at|format_date }}</td>
```

Comment:
These two dates also should only be displayed in DD/MM/YYYY (or localized variants). Time is not relevant either. 
![grafik](https://github.com/FOSSBilling/FOSSBilling/assets/1939311/6b0df612-bbd3-4048-8ea1-0a1a3b164bf4)